include/rocksdb/options.h

```diff
@@ -1427,6 +1427,21 @@ struct DBOptions {
   // of the contract leads to undefined behaviors with high possibility of data
   // inconsistency, e.g. deleted old data become visible again, etc.
   bool enforce_single_del_contracts = true;
+
+  // EXPERIMENTAL
```

Comment:
I think it's safe to say that it's experimental for now, but please let me know if anyone feel this is not necessary.